arcgis-ios-sdk-samples/Content Display Logic/Controllers/OptionsTableViewController.swift

```diff
@@ -31,30 +31,34 @@ class OptionsTableViewController: UITableViewController {
     /// The index of the currently selected option or `nil` if no option is
     /// selected.
     private var selectedIndex: Int?
-    private let onChange: (Int) -> Void
+    private let onChange: (Int?) -> Void
+    private var allowsEmptySelection: Bool
```

Comment:
```suggestion
    private let allowsEmptySelection: Bool
```
as we'll only assign it in the initializers.